using-valtimo/case/case.md

```diff
@@ -0,0 +1,7 @@
+# Case assignee
+
+Case assignee offers a way to configure, per case type, the option to enable set assignee when relevant.
```

Comment:
I think this sentence is a bit much. you try to say the same thing twice. I would change it to something like this: 

```suggestion
It is possible to configure per case type if cases of that type can be assigned to users.
```